.CI/travis_fast_finish.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
```

Comment:
Probably don't want this script going into every feedstock in the event we need to fix it or change it somehow. Maybe it would be better to add it to `conda-forge-build-setup` and then just include the snippet of code to run this script in staged-recipes and the feedstocks.